exams/pearson/constants.py

```diff
@@ -16,3 +17,9 @@
     "EXAMS_SFTP_PASSWORD",
     "EXAMS_SFTP_UPLOAD_DIR",
 ]
+
+# Custom csv dialect for Pearson
+csv.register_dialect(
+    PEARSON_CSV_DIALECT,
+    delimiter='\t',
+)
```

Comment:
does this mean that you configure the `cvs` globally here at application load time?